flatpak_builder_lint/staticfiles/exceptions.json

```diff
@@ -1286,7 +1286,8 @@
         "finish-args-flatpak-spawn-access": "the app predates this linter rule"
     },
     "org.wezfurlong.wezterm": {
-        "finish-args-flatpak-spawn-access": "the app predates this linter rule"
+        "finish-args-flatpak-spawn-access": "required to spawn the user's shell and other commands on the host system; it's the primary purpose of a terminal emulator",
+	"finish-args-unnecessary-xdg-config-access": "required to access host configs, repos, workspaces and so on. https://github.com/flathub-infra/flatpak-builder-lint/issues/270"
```

Comment:
identation is wrong on this one

Reply:
Can you expand on what's wrong? It's consistent with the existing indentation. I edited this through the GitHub web UI and I can't see a difference and it's not clear what is wrong about it. Thanks!

Reply:
![image](https://github.com/flathub-infra/flatpak-builder-lint/assets/62639087/d615f52c-3af0-4220-b756-3e97b3d1f6f6)

Reply:
Fixed!